scripts/translation/qaxml.a.php

```diff
@@ -30,33 +30,35 @@
     if ( implode( "\n" , $s ) == implode( "\n" , $t ) )
         continue;
 
-    $intersect = array_intersect( $s, $t );
-    $onlySource = array_diff( $s , $intersect );
-    $onlyTarget = array_diff( $t , $intersect );
+    $header = true;
+    $match = array();
```

Comment:
You seem to be mixing new and "old" style syntax for creating an array, any reason?

Reply:
No reason. It may reflect the other code style that I worked that day, before writing this code.